### ISSUE TITLE
Updating CUDA version to 10.1 

### ISSUE THREAD

**Issue by benfig1127** [benfig1127/Summer-Research-2020#21]
@mikeizbicki I'm working on implementing the tensorflow model at the moment and it appears that in order for me to use the GPU I must have a newer version of CUDA on the server. I checked the current version of CUDA and this is what I found:
```Cuda compilation tools, release 10.0, V10.0.130```

I believe that the tensorflow 2.2.0 needs CUDA 10.1 in order to work correctly. Here is the current error message I get when trying to use the GPU to calculate the embeddings:
```2020-06-30 14:06:23.072972: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory```

I think this is fixed if you can upgrade the CUDA version, however I cannot do it on my end as it requires root privileges. 

Here is the github issue that suggests installing CUDA 10.1 to fix this error: https://github.com/tensorflow/tensorflow/issues/38578

Let me know if this would indeed resolve the issue, until then I will just set up everything and run some preliminary results on the CPU. Thanks!

**Comment by mikeizbicki** [comment]
I've upgraded cuda for you, so everything should hopefully work now.

**Comment by benfig1127** [comment]
Thanks!

**Comment by benfig1127** [comment]
@mikeizbicki I just check the version on bfigueroa20 and it seems that nothing has been updated. ```Built on Sat_Aug_25_21:08:01_CDT_2018
Cuda compilation tools, release 10.0, V10.0.130```
To clarify, I just re-logged 2 minutes ago so it should have refreshed if it did change. 
Thoughts?

**Comment by mikeizbicki** [comment]
You're right.  The installation hadn't finished yet, but it should be good now.

I've just created a file `use_model.py` in your bfigueroa20b home folder, and
I've verified that it is successfully generating embeddings on the GPU now.

You should be extra careful to specify the `CUDA_AVAILABLE_DEVICES` when working
with tensorflow, because I noticed that it will use all of the GPUs if it's
allowed to, so there won't be anything left for Stefanos.

**Comment by benfig1127** [comment]
Okay, I will make sure that tensorflow is not too selfish and hogs all the resources haha. I find that very entertaining that it just grabs all the GPU's it can. Also I was curious where that model python file came from haha. Also, looks like ive migrated the model correctly and am generating the preliminary embeddings, so we should have something to work with tomorrow!
